docs/variants/msi_z690/overview.md

```diff
@@ -28,23 +29,30 @@ documentation._
 
 ## Useful documents
 
-* [MSI Z690-A PRO DDR4 WiFI site](https://www.msi.com/Motherboard/PRO-Z690-A-WIFI-DDR4)
-* [Board manual](https://download.msi.com/archive/mnu_exe/mb/PROZ690-AWIFIDDR4_PROZ690-ADDR4100x150.pdf)
+* [MSI Z690-A PRO DDR4 WiFI
+  site](https://www.msi.com/Motherboard/PRO-Z690-A-WIFI-DDR4)
+* [Board
+  manual](https://download.msi.com/archive/mnu_exe/mb/PROZ690-AWIFIDDR4_PROZ690-ADDR4100x150.pdf)
 
 ## Press releases
 
 * [PC Gamer](https://www.pcgamer.com/coreboot-on-intel-motherboard/)
 * [Phoronix](https://www.phoronix.com/scan.php?page=news_item&px=Coreboot-Start-ADL-MSI-Dasharo)
-* [Tom's Hardware](https://www.tomshardware.com/news/msi-z690-a-pro-wifi-coreboot)
+* [Tom's
+  Hardware](https://www.tomshardware.com/news/msi-z690-a-pro-wifi-coreboot)
 * [Root.cz](https://www.root.cz/zpravicky/port-coreboot-na-intel-alder-lake-z690-od-msi/)
 * [GN24](https://game-news24.com/2022/04/13/open-source-coreboot-bios-is-ported-to-msi-z690-a-motherboard/)
 * [ITBusiness](https://itbusiness.com.ua/gamezone/113401-u-etogo-neubivaemogo-bios-s-otkrytym-ishodnym-kodom-ogromnyj-potenczial.html)
 
 ## Buy MSI PRO Z690 with Dasharo firmware
 
-You could buy the working station with MSI PRO Z690 motherboard with Dasharo
-firmware from the official
-[3mdeb store](https://3mdeb.com/shop/).
+The working station with MSI PRO Z690 motherboard with Dasharo firmware will
```

Comment:
```suggestion
The working station with MSI PRO Z690-A motherboard with Dasharo firmware will
```

Reply:
I've added the full name of the product.